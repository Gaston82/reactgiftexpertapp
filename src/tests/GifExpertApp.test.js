import React from "react";
import { GifExpertApp } from "../GifExpertApp";
import { shallow } from "enzyme";

describe("Testing <GifExpertApp/>", () => {
  test("should show <GifExpertApp/> correctly", () => {
    const wrapper = shallow(<GifExpertApp />);
    expect(wrapper).toMatchSnapshot();
  });

  test("should show a category list", () => {
    const categories = ["spinetta", "hello"];
    const wrapper = shallow(<GifExpertApp defaultCategories={categories} />);
    //expect(wrapper).toMatchSnapshot();
    //expect(wrapper.find("GifGrid").length).toBe(categories.length);
  });
});
