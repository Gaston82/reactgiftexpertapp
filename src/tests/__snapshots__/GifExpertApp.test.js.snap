// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <GifExpertApp/> should show <GifExpertApp/> correctly 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <Component
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;

exports[`Testing <GifExpertApp/> should show a category list 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <Component
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <Component
      category="spinetta"
      key="spinetta"
    />
    <Component
      category="hello"
      key="hello"
    />
  </ol>
</Fragment>
`;
